ui/dashboards/src/components/Variables/Variable.tsx

```diff
@@ -111,26 +111,27 @@ function ListVariable({ name }: TemplateVariableProps) {
     return computedOptions;
   }, [options, allowAllValue]);
 
+  const valueIsInOptions = useMemo(
+    () =>
+      Boolean(
+        finalOptions.find((v) => {
+          if (allowMultiple) {
+            return (value as string[]).includes(v.value);
+          }
+          return value === v.value;
+        })
+      ),
+    [finalOptions, value, allowMultiple]
+  );
+  const selectValue = valueIsInOptions ? value ?? '' : '';
```

Comment:
I had to use this over `loading` since the value can be set even before the options start loading (via query parameter or default value)